packages/components/search/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@tradeshift/elements.search",
+  "version": "0.4.5",
+  "src": "src/search.js",
+  "main": "lib/search.umd.js",
+  "browser": "lib/search.umd.js",
+  "files": [
+    "lib/*"
+  ],
+  "dependencies": {
+    "@tradeshift/elements": "^0.4.5"
```

Comment:
Please add `@tradeshift/elements.icon` as dependency

Reply:
Good catch! I added it.